zerver/tests/test_events.py

```diff
@@ -996,3 +997,15 @@ def test_use_narrow_if_default_is_none(self):
         self.user_profile.save()
         result = _default_narrow(self.user_profile, [])
         self.assertEqual(result, [])
+
+class AndroidDevGetEmailTest(AuthedTestCase):
+    def test_success(self):
+        # type: () -> None
+        result = self.client.get("/api/v1/dev_android_get_email")
+        self.assert_json_success(result)
```

Comment:
This should probably inspect the result to make sure it has the right format, not just check it returns success.  The idea is that you want the test to detect if the API were to accidentally change, without having to discover that we broke things via the feature no longer working.